README.md

```diff
@@ -23,7 +23,39 @@ See [@Pipeseroni/Packagers][packagers] for list of members who are responsible f
 
 ## Packaging
 
-TBA.
+After a new release is made of a Pipeseroni project, the following fields need
+to be updated:
+
+* `pkgver` in `PKGBUILD`.
+* `sha256sums` in `PKGBUILD` and `.SRCINFO`.
+
+### Automated packaging
+
+The entire packaging process can be automated with [scripts/autoupdate], which
```

Comment:
You should use [relative link](https://blog.github.com/2013-01-31-relative-links-in-markup-files/), I believe the following would work:

```markdown
... with [scripts/autoupdate](scripts/autoupdate), which
```

Reply:
You forgot this one.